src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -836,6 +836,19 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <remarks>https://redis.io/commands/lpop</remarks>
         RedisValue[] ListLeftPop(RedisKey key, long count, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Removes and returns at most specified <paramref name="count"/> of elements from the first non-empty list
+        /// within the set of <paramref name="keys"/> passed into it.
+        /// Starts on the left side of the list.
+        /// If the length of the first non-empty list is less than <paramref name="count"/>, only the elements within that list are returned.
```

Comment:
@slorello89 Does this matter? e.g. is it equally correct and maybe clearer to simplify to:
> Only the elements from the first non-empty list are ever returned, even if <paramref name="count"/> is higher than its length.

...or is that more confusing and/or plain wrong?

Reply:
The important point is that the first list it encounters is what it pops elements out of. Both ways of phrasing it get at that point so whichever is simpler.

Reply:
kk - to me it's confusing currently - will adjust, can always change again!